notebooks/community/experiments/vertex_ai_model_experimentation.ipynb

```diff
@@ -0,0 +1,1750 @@
+{
```

Comment:
<p>This works on workbench, but not Colab. See template for version that works on both.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>I use the last template. Could you point me out to that?</p>

Reply:
Also I tested that notebook on Colab. It works fine.

Reply:
Ahh, I see with the current template there is a sequence that works on both Workbench and Colab.